google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -377,7 +385,12 @@ public void computeDeployments(@NotNull ComputeDeploymentsCallback callback) {
     }
 
     @Override
-    public void disconnect() {
+    public synchronized void disconnect() {
+      // kill any executing process for the current action
+      if (currentAction != null && currentAction.getProcessHandler() != null) {
+        currentAction.getProcessHandler().destroyProcess();
```

Comment:
Yes, and yes. I'll look into how to trigger sigint instead and whether it actually will work.